test/test_mutant_epitope_predictor.py

```diff
@@ -21,6 +21,9 @@
 from topiary import MutantEpitopePredictor, epitopes_to_dataframe
 from varcode import Variant, VariantCollection
 
+ensembl.download()
```

Comment:
It's weird to do the download/index here, what do you think of adding `pyensembl install --releas 75` to the travis config?